### DISCUSSION TITLE
Add new account without downtime

### DISCUSSION THREAD

**Discussion by xorinzor** [docker-mailserver/docker-mailserver#2684] [Q&A] [answered]
Hi,

According to the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/user-management/accounts/) a restart of the container is required in order to apply the changes of adding a new account.

Was wondering if there is any way to do this without downtime of the (SMTP) service (possibly via use of kubernetes?).
The linked issue #552 mentions a reload of postfix would be enough, but the fact that the documentation states a restart of the container is required confuses me a bit as to which source could be considered the truth / best practice.

Thanks!

**Comment by ontheair81** [comment, marked as answer]
You don't need to add accounts manually, you can just use the [setup.sh script](https://docker-mailserver.github.io/docker-mailserver/edge/config/setup.sh/). It automatically reloads the affected services; no need to restart the container.

**Reply by xorinzor** [reply]
Ah, right! I did intend to use the setup.sh script, but didn't realize that the rest of the documentation didn't apply to that.
Good to know, thanks!

**Comment by georglauterbach** [comment]
You can also use the container directly: `docker exec -ti mailserver setup help` should give you an overview :) No need to restart the container.
